lib/net.js

```diff
@@ -1226,14 +1227,18 @@ function setupListenHandle(address, port, addressType, backlog, fd, flags) {
 
     // Try to bind to the unspecified IPv6 address, see if IPv6 is available
     if (!address && typeof fd !== 'number') {
+<<<<<<< HEAD
       rval = createServerHandle('::', port, 6, fd, flags);
+=======
+      rval = createServerHandle(DEFAULT_IPV6_ADDR, port, 6, fd);
+>>>>>>> a6a109ee94... net: some scattered cleanup
```

Comment:
Are these merge conflicts supposed to be here?

Reply:
No, my bad.